server/banned.go

```diff
@@ -44,9 +44,14 @@ func (pi *packetInfo) checkBlameMessage() error {
 		}
 	}
 
-	if validBlame {
+	if !validBlame {
+		return fmt.Errorf("unknown blame reason: %s", packet.GetMessage().GetBlame().GetReason())
+	} else {
```

Comment:
changed logic check: consider this as a bad player if using an invalid blame instead of silencing it.

Reply:
I think this is good.  This ultimately disconnects on invalid reason. The set of reasons is well defined.  If we get an invalid reason it's garbage data. 👍

Reply:
Agreed. This is good. Just need to be careful to add any new blames for the proto here. It also might get a little hairy if certain blame reasons only work for a particular version of the protocol. We can cross that bridge when we get there.